src/NUnitFramework/framework/Internal/RuntimeFramework.cs

```diff
@@ -243,6 +243,7 @@ public static RuntimeFramework CurrentFramework
                                     minor = 0;
                                 }
                             }
+                            key.Close();
```

Comment:
Good catch @Therzok. `GetValue` can throw an exception, so I would prefer to see this handled with a using statement starting at line 229 instead of a call to `Close()`. Would you mind making that change?